modules/kubectl-wrapper/variables.tf

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
We should add a `use_existing_context` variable which will reuse any existing kubectl context and ignore all the other auth variables.

Reply:
added and hence made `project_id`, `cluster_name` etc optional if using `use_existing_context`